base_versions.txt

```diff
@@ -6,8 +6,7 @@ ifaddr==0.1.7
 mediafile==0.8.1
 miniaudio==1.45
 protobuf==4.23.4
-pydantic-extra-types==2.1.0
-pydantic-settings==2.0.2
+pydantic==1.10.13
```

Comment:
Pydantic is currently pinned to 1.10.12 in HA. Not sure how the whole dependency of dependencies song and dance works and if this will be a problem, but just FYI

Reply:
Good catch, I will update that. The versions in this file be translated into ">=", so at least the versions specified here are required.